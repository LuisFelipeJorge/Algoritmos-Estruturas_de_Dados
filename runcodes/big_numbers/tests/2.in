15
sum 234543564645 121
sum 101498473623545 10234586723
sml 1023243454 11111245
big 2243546565765 9
sml 1 1
big 1 10
eql 01 10
eql 11 11
eql 1 10000000000001
eql 123456789123456789 123456789123456789
eql -123456789123456789 -123456789123456789
sum 453534 0000000001
sum 1234567891234567890 98765432198765432100
sum 0924 3210
sml 01 1