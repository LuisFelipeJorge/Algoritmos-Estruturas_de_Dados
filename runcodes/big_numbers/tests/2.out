Resultado 1: 234543564766
Resultado 2: 101508708210268
Resultado 3: 0
Resultado 4: 1
Resultado 5: 0
Resultado 6: 0
Resultado 7: 0
Resultado 8: 1
Resultado 9: 0
Resultado 10: 1
Resultado 11: 1
Resultado 12: 453535
Resultado 13: 100000000089999999990
Resultado 14: 4134
Resultado 15: 0